examples/influxdb-grafana/run-psutil.sh

```diff
@@ -87,22 +93,37 @@ curl --cookie "$COOKIEJAR" \
 	"http://${dm_ip}:3000/api/dashboards/db"
 echo ""
 
-echo -n "starting snapd"
-$SNAP_PATH/bin/snapd --log-level 1 -t 0 --auto-discover $SNAP_PATH/plugin > /tmp/snap.out 2>&1  &
+echo "${green}getting and building snap-plugin-publisher-influxdb${reset}"
+go get github.com/intelsdi-x/snap-plugin-publisher-influxdb
+(cd $SNAP_PATH/../../snap-plugin-publisher-influxdb && make all) || die "Error: failed to get and compile influxdb plugin" 
```

Comment:
For me a happy path run would not restore deps as this will update snap which I often want since it could have pending commits however I added another OR clause that will catch build failure and run a make that includes getting deps.